gui/src/ts/components/node/NodePredicateGrid.tsx

```diff
@@ -0,0 +1,60 @@
+import {NodePageQuery_nodeById_subjectOfEdges} from "api/queries/types/NodePageQuery";
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  List,
+  ListItem,
+  Grid,
+} from "@material-ui/core";
+import * as React from "react";
+import {NodeLink} from "./NodeLink";
+
+const EDGE_PREDICATE_DISPLAY_NAMES: {[index: string]: string} = {};
+
+const EdgeList: React.FunctionComponent<{
+  edges: NodePageQuery_nodeById_subjectOfEdges[];
+  predicate: string;
+}> = ({edges, predicate}) => {
+  let title = EDGE_PREDICATE_DISPLAY_NAMES[predicate];
```

Comment:
TypeScript has a null-coalescing operator like C# now:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

Thanks, Anders.

```
const title = EDGE_PREDICATE_DISPLAY_NAMES[predicate] ?? predicate
```
Should work.

Reply:
awesome